gdpr.module

```diff
@@ -448,6 +448,10 @@ function _get_module_list_by_type($type) {
 function _get_modules() {
   $cache = \Drupal::cache()->get('modules.list');
 
+  if (!$cache) {
+    return [];
```

Comment:
I don't think this is the right approach. If the cache is invalid we should still return a module list.